shared/src/models/utils/start-points.ts

```diff
@@ -7,8 +7,8 @@ import { getCreate } from './get-create';
 export type StartPoint = AlarmGroupStartPoint | TransferStartPoint;
```

Comment:
The input property `startPoint` in `frontend/src/app/pages/exercises/exercise/shared/transfer-overview/start-point-name/start-point-name.component.ts` could use this union type as well.

Reply:
Done